lib/registries/addon/drafts/draft/metadata/route.ts

```diff
@@ -4,17 +4,28 @@ import RouterService from '@ember/routing/router-service';
 import { inject as service } from '@ember/service';
 import DS from 'ember-data';
 
+import DraftRegistrationModel from 'ember-osf-web/models/draft-registration';
 import Analytics from 'ember-osf-web/services/analytics';
+import buildChangeset from 'ember-osf-web/utils/build-changeset';
 
 import { DraftRouteModel } from '../route';
+import MetadataController from './controller';
 
 export default class DraftRegistrationMetadataRoute extends Route {
     @service analytics!: Analytics;
     @service store!: DS.Store;
     @service router!: RouterService;
 
-    model(): DraftRouteModel {
-        return this.modelFor('drafts.draft') as DraftRouteModel;
+    async model(): Promise<DraftRegistrationModel> {
+        const draftRouteModel = this.modelFor('drafts.draft') as DraftRouteModel;
+        const { draftRegistration } = await draftRouteModel.taskInstance;
+        return draftRegistration;
+    }
+
+    setupController(controller: MetadataController, model: DraftRegistrationModel) {
+        this._super();
+        const changeset = buildChangeset(model, {});
+        controller.set('changeset', changeset);
```

Comment:
IMHO, we should build the changeset in the route template using the `changeset` helper:
```hbs
<form>
    <FormControls
        @changeset={{changeset this.draftRegistration}}
        ...attributes as |form|
    >
```